src/dailyai/services/base_transport_service.py

```diff
@@ -220,7 +220,7 @@ async def post_process(post_processor: FrameProcessor):
                 self.interrupt()
                 pipeline_task = asyncio.create_task(pipeline.run_pipeline())
                 started = False
-                continue
+                #continue
```

Comment:
ah, I think you're right that this doesn't need to be there. Want to remove the line entirely?